images/linux/scripts/installers/containers.sh

```diff
@@ -15,6 +15,17 @@ mkdir -p /etc/containers
 echo -e "[registries.search]\nregistries = ['docker.io', 'quay.io']" | tee /etc/containers/registries.conf
 
 # Check that tools are installed properly
-podman --version
-buildah --version
-skopeo --version
+if ! podman --version; then
+    echo "Podman was not installed"
+    exit 1
+fi
+
+if ! buildah --version; then
+    echo "Buildah was not installed"
+    exit 1
+fi
+
+if ! skopeo --version; then
+    echo "Skopeo was not installed"
+    exit 1
+fi
```

Comment:
It's even better to do it through for loop like it's done in the basics.sh 
`https://github.com/actions/virtual-environments/blob/906e2ab86558ca8bbc0087057fa9baa0fd8691f6/images/linux/scripts/installers/basic.sh#L15-L22

Reply:
Thank you, done.